yt/frontends/chimera/io.py

```diff
@@ -0,0 +1,231 @@
+                        a_nuc = np.array(f["abundance"]["a_nuc"])
+                        a_nuc_tile = np.tile(
+                            a_nuc, (xn_c.shape[0], xn_c.shape[1], xn_c.shape[2], 1)
+                        )
+                        yn_c = np.empty(xn_c.shape)
+                        yn_c[:, :, :, :-1] = xn_c[:, :, :, :-1] / a_nuc_tile[:, :, :, :]
+                        yn_c[:, :, :, -1] = xn_c[:, :, :, -1] / a_nuc_rep_c[:, :, :]
+                        ytot = np.sum(yn_c, axis=3)
+                        atot = np.sum(xn_c, axis=3)
+                        abar = np.divide(atot, ytot)
+                        dat_1 = abar[:, :, :].transpose()
+                    elif fname in ("e_rms_1", "e_rms_2", "e_rms_3", "e_rms_4"):
+                        dims = f["mesh"]["array_dimensions"]
+                        n_groups = f["radiation"]["raddim"][0]
+                        n_species = f["radiation"]["raddim"][1]
+                        n_hyperslabs = f["mesh"]["nz_hyperslabs"][()]
+                        energy_edge = f["radiation"]["unubi"][()]
+                        energy_center = f["radiation"]["unui"][()]
+                        d_energy = []
+                        for i in range(0, n_groups):
+                            d_energy.append(energy_edge[i + 1] - energy_edge[i])
+                        d_energy = np.array(d_energy)
+                        e3de = energy_center**3 * d_energy
+                        e5de = energy_center**5 * d_energy
+
+                        psi0_c = f["radiation"]["psi0_c"][:]
+                        row = np.empty(
+                            (n_species, int(dims[2] / n_hyperslabs), dims[1], dims[0])
+                        )
+                        for n in range(0, n_species):
+                            numerator = np.sum(psi0_c[:, :, :, n] * e5de, axis=3)
+                            denominator = np.sum(psi0_c[:, :, :, n] * e3de, axis=3)
+                            row[n][:][:][:] = np.sqrt(
+                                numerator / (denominator + 1e-100)
+                            )
+                            species = int(fname[-1]) - 1
+                            dat_1 = row[species, :, :, :].transpose()
+                    elif fname in (
+                        "lumin_1",
+                        "lumin_2",
+                        "lumin_3",
+                        "lumin_4",
+                        "num_lumin_1",
+                        "num_lumin_2",
+                        "num_lumin_3",
+                        "num_lumin_4",
+                    ):
+                        dims = f["mesh"]["array_dimensions"]
+                        n_groups = f["radiation"]["raddim"][0]
+                        n_hyperslabs = f["mesh"]["nz_hyperslabs"][()]
+                        ergmev = 1.602177e-6
+                        cvel = 2.99792458e10
+                        h = 4.13567e-21
```

Comment:
I wonder if there is a reason not to use unyt here.
```
import unyt as un

ergmev = float((1*un.MeV)/(1*un.erg))
cvel = float(un.c.to("cm/s"))
h = float(un.h.to("MeV * s"))
```

Reply:
Ya, I was going to suggest that too! I don't see a reason not to use it... but it could also be an easy follow-up PR to go through this section and unyt-fy it.